guides/office/kitchen.md

```diff
@@ -0,0 +1,17 @@
+# Kitchen Etiquette
+
+We share the kitchen space with each other as well as with our customers. We have put together a few guidelines for keeping the kitchen nice and tidy.
+
+## Coffee
+
+We love our coffee at Made Tech. We have fresh beans regularly delivered to the office and have multiple thermos drip coffee jugs that keep coffee lovely and fresh. Please ensure:
+
+ - When grinding beans that no meetings are going on in the meeting room as it's very loud and can interrupt voice calls and conversations
+ - When you're done making coffee that you tidy up any spillages so the kitchen is left in a clean state
```

Comment:
can we have a rule that if you take the last cup of coffee in the pot you need to make a new pot? but not if it is later in the day, say after 15.00?

Reply:
_kill the joe, make some mo'_

Reply:
While I'm the biggest fan of the rule (and video on youtube) I don't think we should codify it.